### PR TITLE
Add java -version output

### PR DESCRIPTION
**Pull request by JeromeJu**
Add 'java -version' to the metadata file.

Fixes: adoptium/TKG#260

Signed-off-by: Jerome Ju jeromeqju@gmail.com

### CODE REVIEW

**Comment by JeromeJu** [comment]
What info included in the result of executing 'java -version'shall we kept out of the writing into file?

**Comment by JeromeJu** [comment]
How can I test this before merging?

**Comment by llxia** [comment]
You can test in Grinder using `TKG_OWNER_BRANCH`: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/build?delay=0sec

image

File: src/org/openj9/envInfo/EnvDetector.java [hunk 1]
```diff
@@ -72,6 +74,7 @@ private static void getJavaInfo() {
 			output.write(JDKVERSIONvalue);
 			output.write(JDKIMPLvalue);
 			output.write(JDKVENDORvalue);
+			output.write(JavaVersionValue);
```
Comment (llxia) [inline src/org/openj9/envInfo/EnvDetector.java:77]: We need to output into a different file - AQACert.log

**Comment by llxia** [comment]
@JeromeJu  Could you double-check where the file is output to? It is expected this log file is output under TKG dir https://github.com/adoptium/aqa-tests/pull/3316. Could you also provide the Grinder link? Thanks

**Comment by JeromeJu** [comment]
I tried running the Grinder test earlier but not fully sure how to check the file output on Jenkins.
Grinder link [#3539](https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3539/console)

Noticed that the `No artifacts found that match the file pattern "aqa-tests/TKG/AQACert.log". Configuration error?` warning is gone. So I guessed it shall output to the correct path? as discussed in https://github.com/adoptium/aqa-tests/pull/3316.

**Comment by llxia** [comment]
> So I guessed it shall output to the correct path?

yes, just add `pwd` and see where is your current dir. 
@renfeiw any variable we can directly use to get to TKG dir?

**Comment by llxia** [comment]
Thanks @JeromeJu . The file is generated at the correct place.

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
thanks @JeromeJu !

**Pull request merged**